src/pages/planters/[planterid].js

```diff
@@ -36,6 +36,20 @@ import { useMapContext } from '../../mapContext';
 import { makeStyles } from '../../models/makeStyles';
 import * as utils from '../../models/utils';
 
+const mapContinentName = (continentName) => {
+  if (!continentName) return undefined;
+
+  const nameArray = continentName.toLowerCase().split(' ');
+  if (nameArray.length > 1) {
+    return nameArray[0][0] + nameArray[1][0];
+  }
+  if (nameArray.length === 1) {
+    return nameArray[0][0] + nameArray[0][1];
+  }
+
+  return undefined;
+};
+
```

Comment:
I suggest to move this into the component, so this become the component's responsibility to handle the `full-name`, and also support the abbreviation too.
In this way, we can reuse the code, what do you think?

Reply:
do u mean to move the function mapContinentName into CustomWorldMap Component @dadiorchen

Reply:
Yes, @Mohmn exactly.